resources/js/app.js

```diff
@@ -12,16 +32,15 @@
 		})
 	})
 
-	app.controller('TabController', function () {
-		this.tab = 1
-		this.setTab = function (stab) {
-			this.tab = stab
-			$(".button-collapse").sideNav('hide');
-		}
-		this.isSet = function (stab) {
-			return this.tab == stab
-		}
-	})
+    app.controller('TabController', function ($location) {
+        this.setTab = function (stab) {
+            $location.path(stab);
```

Comment:
In the setTab function, we will have to set this.tab as selected tab, so that the indicator moves to selected tab, 
```
this.tab = stab
$location.path(stab);
```